### DISCUSSION TITLE
Single camera, multiple previews

### DISCUSSION THREAD

**Discussion by situam** [raspberrypi/picamera2#428] [Q&A]
Is it possible to render two previews (QTGL windows, for example) from a single camera source? Something which functions like mmal's 'splitting' component would solve this (see an example here: https://forums.raspberrypi.com/viewtopic.php?t=232991#p1582997)

Application details:
One camera output should be displayed fullscreen on two displays (1x Hyperpixel4 DPI 800x480px, 1x HDMI projector 1920x1080px) with acceptable framerate (>15fps, ideally >24fps). I'm using a Raspberry Pi 3B+ running the latest Bullseye release. So far have managed to get a QTGL preview to show on both screens when displays are configured as overlapped, but with different resolutions this is suboptimal, as well as an OpenCV imshow()-based script which copies the image array to two windows, but this is much too slow.

**Comment by davidplowman** [comment]
Hi, I'm afraid this isn't supported. You should find that you can run one QTGL preview (I would recommend the large one), and you could display another with OpenCV, for example. For best performance, avoiding X Windows and using DRM/KMS would almost certainly work better, but you'd have to work out how to drive those APIs which is tricky if you're not familiar with them.
